src/index.html

```diff
@@ -300,10 +320,8 @@ <h3>More options</h3>
           <span>Permissions:</span>
           <br />
           <select name="ohyaaah" id="perms" style="width: 95%;" required>
```

Comment:
ohyahhhhhhh??

Reply:
@Chuppay stop wasting everyone's time with this garbage please